src/NServiceBus.Core/Transports/Msmq/MsmqMessageSender.cs

```diff
@@ -50,7 +51,9 @@ public void Send(TransportMessage message, Address address)
                                 new MessageQueue(MsmqUtilities.GetReturnAddress(message.ReplyToAddress.ToString(),
                                                                                 address.ToString()));
 
-                        if (UnitOfWork.HasActiveTransaction())
+                        bool suppressNativeTransactions;
```

Comment:
Question - is there a situation when we want to allow for native transactions other than when dtc is enabled? If not then could we check for dtc enabled condition instead of using pipeline?
Or it won't be exactly the same effect?